cvcreator/vitae/schema.py

```diff
@@ -161,6 +161,49 @@ class MetaInformation(StrictModel):
     nationality_image: str = "nationality"
 
 
+class SectionTitles(StrictModel):
+    """Titles of the different sections. In English by default."""
+
+    professional_experience: str = "Professional Experience"
+    education: str = "Education"
+    technical_skills: str = "Technical Skills"
+    languages: str = "Languages"
+    personal_skills: str = "Personal Skills"
+    hobbies: str = "Interests and hobbies"
+    projects: str = "Extended description of selected projects"
```

Comment:
Suggestion to have capitalization in these titles as well

```suggestion
    hobbies: str = "Interests and Hobbies"
    projects: str = "Extended Description of Selected Projects"
```